packages/delivery-x-domain-request/delivery.js

```diff
@@ -3,14 +3,22 @@ const payload = require('@bugsnag/core/lib/json-payload')
 module.exports = (client, win = window) => ({
   sendEvent: (event, cb = () => {}) => {
     const url = getApiUrl(client._config, 'notify', '4', win)
+    const body = payload.event(event, client._config.redactedKeys)
+
     const req = new win.XDomainRequest()
     req.onload = function () {
       cb(null)
     }
+    req.onerror = function () {
+      client._logger.error('Event failed to send…\n')
```

Comment:
node does `logger.error(`Event failed to send…` so I thought maybe the browser delivery methods should too